### PR TITLE
clean up the readme

### PR DESCRIPTION
**Pull request by webbrowser11**
ye ye

### CODE REVIEW

**Comment by BDadmehr0** [comment]
I suggest adding an example for the input path and the expected output structure to make it clearer for users.

File: README.md [hunk 1]
```diff
@@ -9,11 +9,14 @@ pip install os-sys
...
-# how to use
-click on the .py and type in the folder you want to organize with the full path.
-# result
-you have folders in the folder you just organized named the file extension of its files.
+> for windows machines: theres an exe download for windows in the each latest release
+## how to use
+### (windows):
+run the exe and then in the cmd that pops up enter the a full location of the folder you want to organize.
+### (non-windows):
+unzip the folder and run the .py file and then in the cmd that pops up enter the a full location of the folder you want to organize.
```
Comment (BDadmehr0) [inline README.md:20]: It is better to write words like .py, .exe inside '`words`'.

File: README.md [hunk 2]
```diff
@@ -9,11 +9,14 @@ pip install os-sys
...
-> theres an exe download in the 1.0 release
-# how to use
-click on the .py and type in the folder you want to organize with the full path.
-# result
-you have folders in the folder you just organized named the file extension of its files.
+> for windows machines: theres an exe download for windows in the each latest release
+## how to use
+### (windows):
+run the exe and then in the cmd that pops up enter the a full location of the folder you want to organize.
+### (non-windows):
```
Comment (BDadmehr0) [inline README.md:19]: non-windows !?

Mac or Linux, or Unix-based systems

**Review by BDadmehr0 — CHANGES_REQUESTED** [review]

**Comment by webbrowser11** [comment]
thanks, i will propose to add some of this stuff

**Comment by webbrowser11** [comment]
@BDadmehr0 if you go to my repo on that branch and make a pr with these changes, i will accept it and it will be merged here.

**Comment by webbrowser11** [comment]
if you want to wait and let me do this therefor make the development time to clean up the readme longer, you can do that too.

**Comment by BDadmehr0** [comment]
> @BDadmehr0 if you go to my repo on that branch and make a pr with these changes, i will accept it and it will be merged here.

There's no rush; make the changes whenever you have time.

**Comment by webbrowser11** [comment]
i was saying you could do it too, on my repo, so it gets merged to this repo.

**Comment by webbrowser11** [comment]
u would be contrib too

**Comment by BDadmehr0** [comment]
> u would be contrib too


The problem is that it's currently nighttime in Iran, and I have shut down my system. I won't be able to make any changes for the next 20 hours. If the changes are not made by then, I will take care of it myself.

**Comment by webbrowser11** [comment]
k

**Comment by cWorksLLC** [comment]
Done. I like the changes here.

**Comment by webbrowser11** [comment]
nice, might come back to do more tmrw

**Pull request merged**